packages/web-components/fast-components/src/tree-item/tree-item.styles.ts

```diff
@@ -94,233 +101,198 @@ export const treeItemStyles: (
-        background: none;
-        border: none;
-        outline: none;
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        width: calc((${expandCollapseButtonSize} + (${designUnit} * 2)) * 1px);
-        height: calc((${expandCollapseButtonSize} + (${designUnit} * 2)) * 1px);
-        padding: 0;
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        cursor: pointer;
-        margin-left: 6px;
-        margin-right: 6px;
-    }
+        .expand-collapse-button svg {
+            transition: transform 0.1s linear;
+            pointer-events: none;
+        }
 
-    .expand-collapse-glyph {
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        width: 16px;
-        height: 16px;
-        transition: transform 0.1s linear;
+        .start,
+        .end {
+            display: flex;
+        }
 
-        pointer-events: none;
-        fill: currentcolor;
-    }
+        .start {
+            /* need to swap out once we understand how horizontalSpacing will work */
+            margin-inline-end: calc(${designUnit} * 2px + 2px);
+        }
 
-    .start,
-    .end {
-        display: flex;
-        fill: currentcolor;
-    }
+        .end {
+            /* need to swap out once we understand how horizontalSpacing will work */
+            margin-inline-start: calc(${designUnit} * 2px + 2px);
+        }
 
-    ::slotted(svg) {
-        /* TODO: adaptive typography https://github.com/microsoft/fast/issues/2432 */
-        width: 16px;
-        height: 16px;
-    }
+        :host(.expanded) > .items {
+            display: block;
+        }
 
-    .start {
-        /* TODO: horizontalSpacing https://github.com/microsoft/fast/issues/2766 */
-        margin-inline-end: calc(${designUnit} * 2px + 2px);
-    }
+        :host([disabled]) {
+            opacity: ${disabledOpacity};
+            cursor: ${disabledCursor};
+        }
 
-    .end {
-        /* TODO: horizontalSpacing https://github.com/microsoft/fast/issues/2766 */
-        margin-inline-start: calc(${designUnit} * 2px + 2px);
-    }
+        :host(.nested) .content-region {
+            position: relative;
+            margin-inline-start: var(--expand-collapse-button-size);
+        }
 
-    :host([expanded]) > .items {
-        display: block;
-    }
+        :host(.nested) .expand-collapse-button {
+            position: absolute;
+        }
 
-    :host([disabled]) .content-region {
-        opacity: ${disabledOpacity};
-        cursor: ${disabledCursor};
-    }
+        :host(.nested) .expand-collapse-button:hover {
+            background: ${expandCollapseHoverBehavior};
+        }
 
-    :host(.nested) .content-region {
-        position: relative;
-        margin-inline-start: var(--expand-collapse-button-size);
-    }
+        :host(:not([disabled])[selected]) .positioning-region {
+            background: ${neutralFillStealthActive};
```

Comment:
`Active` should only be used for the `active` state. Let's follow the selected state update we did for listbox option here for the same accessibility reason.

Reply:
combined this selector with the active selector